actionview/lib/action_view/layouts.rb

```diff
@@ -405,11 +405,11 @@ def _normalize_layout(value)
     #
     # ==== Parameters
     # * <tt>formats</tt> - The formats accepted to this layout
-    # * <tt>require_layout</tt> - If set to true and layout is not found,
-    #   an +ArgumentError+ exception is raised (defaults to false)
+    # * <tt>require_layout</tt> - If set to +true+ and layout is not found,
+    #   an +ArgumentError+ exception is raised (defaults to +false+)
```

Comment:
When we configure stuff or document default values, unless the underlying code is doing a `== false` and the value has to be exactly that, we document just a flag. That is, the method does not _require_ that users pass singletons, it just relies on standard boolean semantics, so we do not document singletons. We document just what the method needs.

For example, if the value comes from something dynamic, users can spare a `!!` to make sure the call satisfies the contract, they just pass the value with boolean semantics down.